webplatformcompat/jinja2/webplatformcompat/feature-basic.html

```diff
@@ -93,7 +93,7 @@
           {%- set version_id = support['links']['version'] %}
           {%- set version = collection['versions'][version_id] %}
           {%- if not loop.first %}<br>{% endif %}
-          {%- if version.version %}
+          {%- if version.version != 'current' %}
```

Comment:
nit-pick: Could clean up `if` conditional lines' indentation to make the logic clearer?